src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let wasteStyles = sourceString.split(';');
+  const obj = {};
+
+  wasteStyles = wasteStyles.map((line) => line.trim());
+
+  for (const styles of wasteStyles.filter(Boolean)) {
+    const style = styles.split(':');
+    const key = style[0].trimStart().trimEnd();
+    const value = style[1].trimStart().trimEnd();
```

Comment:
The use of `trimStart()` and `trimEnd()` can be replaced with `trim()` for both `key` and `value` to simplify the code. `trim()` removes whitespace from both ends of a string.